docs/vscode/lobster/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+/**@type {import('eslint').Linter.Config} */
+// eslint-disable-next-line no-undef
+module.exports = {
```

Comment:
why is this stuff in this dir and not in LSP? same for the files below

Reply:
I forgot, fixed